api/Location.json

```diff
@@ -361,10 +361,10 @@
               "notes": "Intranet sites are set to Compatibility View, which will emulate IE7 and omit <code>window.location.origin</code>."
             },
             "opera": {
-              "version_added": "10"
```

Comment:
This came from https://github.com/mdn/browser-compat-data/pull/4209 and looks like it was based on real testing. http://mdn-bcd-collector.appspot.com/tests/api/Location/origin does look good, but can you dig into this and see if there's something to the Opera 10 claim?

Reply:
I'm not sure how they obtained Opera 10 in their results; it is `undefined` in both Opera 10.6 and 12.1.  My guess is that they must have performed a test while on a page that implemented a polyfill.

Reply:
That might be. I'll leave a comment on that PR just in case, but thanks for testing further!